include/deal.II/lac/trilinos_tpetra_vector.h

```diff
@@ -1003,65 +1003,86 @@ namespace LinearAlgebra
                                      const size_type *indices,
                                      const Number    *values)
     {
+      // First create an alias for the type of a view into our vectors.
+      // The actual type is declared through several re-directions in
+      // Tpetra, so instead of spelling it out, we get it via decltype:
 #  if DEAL_II_TRILINOS_VERSION_GTE(13, 2, 0)
-      auto vector_2d_local = vector->template getLocalView<Kokkos::HostSpace>(
-        Tpetra::Access::ReadWrite);
-      auto vector_2d_nonlocal =
+      using ViewType2d =
+        decltype(vector->template getLocalView<Kokkos::HostSpace>(
+          Tpetra::Access::ReadWrite));
+      ViewType2d vector_2d_local =
+        vector->template getLocalView<Kokkos::HostSpace>(
+          Tpetra::Access::ReadWrite);
```

Comment:
I prefer
```suggestion
      auto vector_2d_local =
        vector->template getLocalView<Kokkos::HostSpace>(
          Tpetra::Access::ReadWrite);
      using ViewType2d = decltype(vector_2d_local);    
```
and similarly below.

Reply:
Yes, good point. I fixed it in both places.